<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthenticationForm</class>
 <widget class="QWidget" name="AuthenticationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Authentication</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>891</width>
     <height>521</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="txtLoginUsername">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblLoginUsername">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLoginPassword">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLoginPassword">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblLoginSection">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Login Section</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSignupPassword">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSignupPassword">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSignupUsername">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSignupSection">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Signup Section</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSignupUsername">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFirstname">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSignupFirstname">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblLastname">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSignupLastname">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>21</width>
      <height>441</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogout">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSignup">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>311</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Signup</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblResult">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>470</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>751</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
